src/components/HeroHeader/index.jsx

```diff
@@ -24,25 +25,19 @@ const HeroHeader = ({ t, client, breakpoints: { isMobile } }) => {
   const { data: instanceSettings } = useInstanceSettings(client)
   const publicName = get(instanceSettings, 'public_name', '')
 
+  const showLogout =
+    flag('home_corner_logout') === null ? true : flag('home_corner_logout')
+  const showSettings =
+    flag('home_corner_settings') === null ? false : flag('home_corner_settings')
```

Comment:
To flags, I'd prefer if we'd use the `namespace.subnamespace.flag-name` naming scheme. In harvest, we used `harvest.bi-konnector-policy` for example.

home_corner_settings -> home.corner.settings ?

Reply:
Premonition 😅